frontend/src/interfaces/action.ts

```diff
@@ -19,10 +19,7 @@ export interface IMatch {
 }
 
 export interface IAction {
-    id: number;
     action: string;
     sortOrder: number;
     executionParams: Record<string, unknown>;
-    createdAt: string;
-    createdByUserId: number;
```

Comment:
Now I understand these changes a bit better. 

I think these interfaces should still reflect our models that we're fetching from the DB. If we're saying that `useActions` returns `IActionSet[]`, which includes `actions: IAction[]` where `IAction` does not include some of the data that we're returning, we're not really being honest in our types.

I think the UI should adapt to this and not the other way around, so I would revert this.